### PR TITLE
Display the new op info structures.

### PR DESCRIPTION
**Pull request by fmalita**
After http://crrev.com/942533002/, SkiaBenchmarkingExtension op info is no longer a flat string - but an object hierarchy. There are probably much better ways to present it, but this is a quick hack to display the information.

### CODE REVIEW

File: trace_viewer/extras/cc/picture_ops_list_view.html [hunk 1]
```diff
@@ -90,12 +90,13 @@
           item.appendChild(elementInfo);
         }
 
-        // Display each of the Skia ops.
-        op.info.forEach(function(info) {
+        // Display the Skia params.
+        // FIXME: now that we have structured data, we should format it.
```
Comment (dj2) [inline trace_viewer/extras/cc/picture_ops_list_view.html]: Can you file a bug for this and link it here?

File: trace_viewer/extras/cc/picture_ops_list_view.html [hunk 2]
```diff
@@ -90,12 +90,13 @@
...
+        if (op.info.length > 0) {
           var infoItem = document.createElement('div');
-          infoItem.textContent = info;
+          infoItem.textContent = JSON.stringify(op.info);
           item.appendChild(infoItem);
-        });
```
Comment (dj2) [inline trace_viewer/extras/cc/picture_ops_list_view.html]: What was the ); from? Do we not need it anymore?

Comment (fmalita) [inline trace_viewer/extras/cc/picture_ops_list_view.html]: Closing for forEach above (now gone).

Comment (dj2) [inline trace_viewer/extras/cc/picture_ops_list_view.html]: Ah, missed that. Thanks.

**Comment by dj2** [comment]
lgtm once the bug link is in.

File: trace_viewer/extras/cc/picture_ops_list_view.html (hunk 1 above)
Comment (fmalita) [inline trace_viewer/extras/cc/picture_ops_list_view.html]: Done.

**Pull request merged**